pkg/cloud/volume_limits.go

```diff
@@ -262,6 +271,14 @@ var nvmeInstanceStoreVolumes = map[string]int{
 	"x2gd.12xlarge":  2,
 	"x2gd.16xlarge":  2,
 	"x2gd.metal":     2,
+	"x2idn.large":    1,
+	"x2idn.xlarge":   1,
+	"x2idn.2xlarge":  1,
+	"x2idn.4xlarge":  1,
+	"x2idn.8xlarge":  1,
+	"x2idn.16xlarge": 1,
+	"x2idn.32xlarge": 2,
```

Comment:
Can you also add `x2idn.24xlarge` (correct limit is 2)? Otherwise lgtm

Reply:
Are `x2idn.8xlarge`, `x2idn.4xlarge`, `x2idn.2xlarge`, `x2idn.xlarge`, `x2idn.large` valid? 

https://aws.amazon.com/ec2/instance-types/x2i/ lists `x2idn.metal`, `x2idn.16xlarge`, `x2idn.24xlarge`, `x2idn.32xlarge`.

```
$ aws --region us-east-1 ec2 describe-instance-types | jq '.InstanceTypes | map(.InstanceType)' | grep x2idn
  "x2idn.24xlarge",
  "x2idn.16xlarge",
  "x2idn.metal",
  "x2idn.32xlarge",
```

Reply:
I don't mind if we remove them and just keep the x2idn.16xlarge and x2idn.32xlarge.
Would you like me to amend my PR?

Reply:
Yes, if possible, please amend your PR to drop all the invalid instances (`x2idn.8xlarge` and below) and add `x2idn.24xlarge` with limit 2 (which is valid and is missing).

Reply:
Fixed